palace/utils/mfemcoefficients.hpp

```diff
@@ -117,37 +116,35 @@ class BdrChargeCoefficient : public mfem::Coefficient, public BdrGridFunctionCoe
 private:
   const mfem::ParGridFunction &E;
   const MaterialOperator &mat_op;
+  mfem::Vector C1, W, VU, VL, nor;
```

Comment:
Maybe mark as `mutable` given these are internal working variables, their `const` status needn't block making the class `const`.

Reply:
Done in [b26aead](https://github.com/awslabs/palace/pull/24/commits/b26aead1d3858a4bf54900a877b3b273a2228a14).